cdn_lambda/functions/map_to_s3/map_to_s3.json

```diff
@@ -2,5 +2,9 @@
   "table": {
     "name": "exodus",
     "region": "us-east-1"
-  }
+  },
+  "uri_aliases": [
```

Comment:
I just want to mention, in case it wasn't clear, that I plan to do a later change which removes the aliases from this repo and uses cdn-definitions package instead.

Reply:
Yeah, I'd  prefer that as well.